### PR TITLE
Add files via upload

### LINKED ISSUE #16846: UI style errors: Menu items capitalization and … (ellipses) mark misuse
### Windows Terminal version

1.20.10572.0

### Windows build number

10.0.22631.3235

### Other Software

_No response_

### Steps to reproduce

Display the tab contextual menu, preferably with a second pane open to show all options.
image
There are several problems with this menu.

### First: Excessive capitalization
If you check out the style guide: https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-common-text-elements
You'll see that `UI text or strings` should use `Sentence-style capitalization`, this is further described at https://learn.microsoft.com/en-us/style-guide/capitalization
```
capitalize the first word and lowercase the rest.
Exceptions: Proper nouns, including brand, product, and service names, are always capitalized.
If a title or heading includes a colon, capitalize the first word after it.
```
`Tab`, `New`, `Window`, `Right` are not brands, products, or service names, they should not be capitalized.
This looks worse in other languages as determiners make the capitalized words lose their vertical alignment, as seen in #16819.

### Second: Misuse of `…` (ellipses)
From the Windows UX design guide: https://learn.microsoft.com/en-us/windows/win32/uxguide/text-ui#guidelines
```
Ellipses mean incompleteness. Use ellipses in UI text as follows:

Commands: Indicate that a command needs additional information.
Don't use an ellipsis whenever an action displays another window only when additional information is required.
For more information, see [Command Buttons](https://learn.microsoft.com/en-us/windows/win32/uxguide/ctrl-command-buttons).

(… more details about truncated text, use in data and labels not included here)
```
I know this is getting out of style, but `…` and `>` next to menu items were originally designed to provide predictability.
The `…` after a menu item means selecting it will display a follow-up dialog or window asking for more options before completing the selected action, while `>` after a menu item means selecting it will display a cascading submenu, finally, neither means selecting it will perform the action immediately.
Therefore, they are mutually exclusive, you cannot have `…` and `>` on the same menu item, clicking it either displays a dialog/window, or a submenu, it cannot do both.

More modern Windows UI is trying to get rid of `…` on menu items, trading discoverability and predictability for the sake of visual style, but including it when it doesn't mean a subsequent window will get displayed breaks both the UI guidelines we had since Windows 3.x, and the modern look. Those `Close…` have to go.

Note the tab contextual menu was taken as an example because it contains all the problems I wanted to mention, but other menus also contain similar errors.

### Expected Behavior

UI consistency _(didn't there used to be a UI guidelines check before inclusion as a Windows component?)_.

### Actual Behavior

Unpredictability and confusion 🤪

### PR DESCRIPTION
**Pull request by Ismith507**
## Summary of the Pull Request
Edited Resources.resw to reflect the style guidlines
## References and Relevant Issues
Issue #16846 
## Detailed Description of the Pull Request / Additional comments
Edited Resources.resw to reflect the desired style guidelines. Specifically sentence style capitalization and use of elipses.
## PR Checklist
- [x] Closes #16846
- [ ] Tests added/passed
- [ ] Documentation updated
   - If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/terminal) and link it here: #xxx
- [ ] Schema updated (if necessary)

### CODE REVIEW

File: src/cascadia/TerminalApp/Resources/en-US/Resources.resw [hunk 1]
```diff
@@ -187,6 +187,9 @@
   <data name="Quit" xml:space="preserve">
     <value>Quit</value>
   </data>
+  <data name="CloseWindowWarningTitle" xml:space="preserve">
```
Comment (zadjii-msft) [inline src/cascadia/TerminalApp/Resources/en-US/Resources.resw:190]: This resource seems to not be used anywhere?

File: src/cascadia/TerminalApp/Resources/en-US/Resources.resw [hunk 2]
```diff
@@ -215,10 +218,10 @@
     <value>Web search</value>
   </data>
   <data name="TabColorChoose" xml:space="preserve">
-    <value>Change tab color</value>
+    <value>Color...</value>
```
Comment (zadjii-msft) [inline src/cascadia/TerminalApp/Resources/en-US/Resources.resw:221]: isn't this literally the opposite of what was done in #16886

File: src/cascadia/TerminalApp/Resources/en-US/Resources.resw [hunk 3]
```diff
@@ -215,10 +218,10 @@
     <value>Web search</value>
   </data>
   <data name="TabColorChoose" xml:space="preserve">
-    <value>Change tab color</value>
+    <value>Color...</value>
   </data>
   <data name="TabColorCustomButton.Content" xml:space="preserve">
-    <value>Custom</value>
+    <value>Custom...</value>
```
Comment (zadjii-msft) [inline src/cascadia/TerminalApp/Resources/en-US/Resources.resw:224]: (and here too)

File: src/cascadia/TerminalApp/Resources/en-US/Resources.resw [hunk 4]
```diff
@@ -835,7 +835,7 @@
     <value>Close this tab</value>
   </data>
   <data name="NewTabMenuFolderEmpty" xml:space="preserve">
-    <value>Empty</value>
+    <value>Empty...</value>
```
Comment (zadjii-msft) [inline src/cascadia/TerminalApp/Resources/en-US/Resources.resw:838]: (and here too)

**Review by zadjii-msft — CHANGES_REQUESTED** [review]

**Comment by DHowett** [comment]
> "the opposite of ..."

I think what happened here is that lsmith took a snapshot of the file as it was when they started working, and re-uploaded it via the GitHub web UI after they made their modifications.

If that's the case - there are some changes that need to be integrated from #16886!

**Comment by Ismith507** [comment]
Hi, I can integrate changes from #16886 and reupload! After that, is there anything else to be done before I can merge? or does #16886 fix the problem and is this PR now redundant?

**Comment by zadjii-msft** [comment]
Okay we looked over it all again - looks like #16886 took care of things here. Thanks anyways!

**Pull request closed without merging**